rollup.config.js

```diff
@@ -100,7 +117,8 @@ const getRollupConfig = ({ pwd, babelConfigFile }) => {
     external,
     input,
     output: {
-      dir: `${SOURCE_DIR}/${pkgConfig.main.replace('/index.js', '')}`,
```

Comment:
We do not want to hard code the dir here because someone might want to configure a different directory for these files.